.github/workflows/build-and-push-assets.yml

```diff
@@ -53,6 +53,21 @@ on:
         type: string
         default: ''
         required: false
+      INSTALL_WP_CLI:
+        description: Whether to Install WP cli or not.
+        default: false
+        required: false
+        type: boolean
+      PHP_VERSION:
+        description: PHP version with which the WP CLI is going to be executed.
+        default: "8.1"
+        required: false
+        type: string
+      BRANCHES_USING_COMPILE_SCRIPT_PROD:
+        description: Branches that should use the COMPILE_SCRIPT_PROD script
+        default: '[]'
+        required: false
+        type: string
```

Comment:
Isn't this something whcih should go into the calling workflow via `on.push.branches` ? https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushbranchestagsbranches-ignoretags-ignore